homeassistant/components/coinbase/const.py

```diff
@@ -85,6 +87,7 @@
     "CZK": "CZK",
     "DAI": "DAI",
     "DASH": "DASH",
+    "DDX": "DDX",
```

Comment:
```suggestion
```

Derivadao (DDX) is not tradable on Coinbase and so is not valid as a wallet, https://www.coinbase.com/price/derivadao